packages/block-editor/src/components/inner-blocks/README.md

```diff
@@ -131,36 +131,36 @@ If locking is not set in an `InnerBlocks` area: the locking of the parent `Inner
 
 If the block is a top level block: the locking of the Custom Post Type is used.
 
-### `renderAppender`
+### `appender`
 
--   **Type:** `Component|false`
--   **Default:** - `undefined`. When `renderAppender` is not specified, the default appender is shown. If a `false` value is provided, no appender is rendered.
+-   **Type:** `Object|null`
+-   **Default:** - `undefined`. When `appender` is not specified, the default appender is shown. If a `null` value is provided, no appender is rendered.
 
-A component to show as the trailing appender for the inner blocks list.
+A instantiated component to show as the trailing appender for the inner blocks list.
 
 #### Notes
 
 -   For convenience two predefined appender components are exposed on `InnerBlocks` which can be used for the prop:
-    -   `InnerBlocks.ButtonBlockAppender` - display a `+` (plus) icon button as the appender.
-    -   `InnerBlocks.DefaultBlockAppender` - display the default block appender, typically the paragraph style appender when the paragraph block is allowed.
+    -   `<InnerBlocks.ButtonBlockAppender />` - display a `+` (plus) icon button as the appender.
+    -   `<InnerBlocks.DefaultBlockAppender />` - display the default block appender, typically the paragraph style appender when the paragraph block is allowed.
 -   Consumers are also free to pass any valid component. This provides the full flexibility to define a bespoke block appender.
```

Comment:
I think this line will need to be updated to say 'any valid react element' or 'any instantiated component'.

Reply:
Also in https://github.com/WordPress/gutenberg/pull/29911/commits/25317c9352acca0c96e7ef004ef721616a5f8bda